src/models/GOKU.jl

```diff
@@ -101,11 +139,30 @@ function sample(μ::T, logσ²::T, model::LatentDiffEqModel{GOKU}) where T <: Tu
     return ẑ₀, θ̂
 end
 
+@doc raw"""
+    default_layers(model_type, input_dim::Int, diffeq;
+        device = cpu,
+        hidden_dim_resnet = 200, rnn_input_dim = 32,
+        rnn_output_dim = 16, latent_dim = 16,
+        latent_to_diffeq_dim = 200, θ_activation = softplus,
+        output_activation = σ, init = Flux.kaiming_uniform(gain = 1/sqrt(3)))
+Generates default encoder and decoder layers that are to be fed into the LatentDiffEqModel.
+# Arguments
+model\_type: GOKU() or LatentODE()\
+input\_dim: Dimension of input, 28*28 for pendulum data provided\
+diffeq: Differential equation, e.g. Pendulum() for GOKU or NODE(16) for LatentODE
```

Comment:
Here I would give more details, such as:

Differential equations structure, containing fields `prob`, `solver` and `sensealg`, which correspond to [DifferentialEquations.jl](https://diffeq.sciml.ai/dev/)'s problem, solver and [sensitivity algorithm](https://diffeqflux.sciml.ai/dev/ControllingAdjoints/), respectively. For an example, see https://github.com/gabrevaya/LatentDiffEq.jl/blob/master/examples/pendulum_friction-less/pendulum.jl

Reply:
I would add this description of the diffeq layer to the Decoder function too.